package.json

```diff
@@ -2,46 +2,103 @@
   "name": "videojs-contrib-media-sources",
   "version": "2.4.3",
   "description": "A Media Source Extensions plugin for video.js",
-  "main": "videojs-media-sources.js",
+  "author": "Brightcove",
+  "license": "Apache-2.0",
+  "main": "src/plugin.js",
+  "browser": "src/plugin.js",
+  "keywords": [
+    "videojs",
+    "videojs-plugin"
+  ],
+  "browserify": {
+    "transform": [
+      "browserify-shim",
+      "babelify"
```

Comment:
The reason this isn't included in here in the generator's output is that the Babel to Browserify process is two steps because we want the `es5/` directory to exist for users who are not using Babel.